prow/cmd/gcsupload/README.md

```diff
@@ -0,0 +1,38 @@
+# `gcsupload`
+
+`gcsupload` uploads artifacts to cloud storage at a path resolved from the job configuration.
+
+`gcsupload` can be configured by either passing in flags or by specifying a full set of options
+as JSON in the `$GCSUPLOAD_OPTIONS` environment variable, which has the following form:
+
+```json
+{
+    "bucket": "kubernetes-jenkins",
+    "sub_dir": "",
+    "items": [
+        "/logs/artifacts/"
+    ],
+    "path_strategy": "legacy",
```

Comment:
Can the example use explicit? Also can it default to this if it isn't already set?

Reply:
Wanted to use `legacy` so we would need the `default_{org,repo}` fields in the example too

Reply:
The "strongly advised to use explicit" feels more like a parenthetical here than something we believe in.

Can the first example please use explicit? Personally I wouldn't mind making the default_org, default_repo, path_strategy hard to read so that people use them. Alternatively add a second example that uses them?